app/serializers/event_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class EventSerializer < ActiveModel::Serializer
```

Comment:
Before I move onto making the specs more robust what do you think about this set up?  I decided to give ActiveRecord Serialization a try.  Definitely makes small files.  Down side is I can't really customize which fields we take from a model for each payload.